recipes/slow5tools/0.7.0/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "slow5tools" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hasindu2008/slow5tools/releases/download/v{{ version }}/slow5tools-v{{ version }}-release.tar.gz
+  sha256: a38ea6e0e1c647b6d2b333d15201d9ce5932dda52ec87537e6d982ba1bfbb595
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - zlib
+    - hdf5
+    - wget
+  run:
```

Comment:
I think you can remove both from the run section. They are automatically exported by conda.